LondonBritishLibrary/add/BLadd26553.xml

```diff
@@ -0,0 +1,172 @@
+                        <summary/>
+
+                        <msItem xml:id="ms_i1">
+                          <locus from="6r"></locus>
+                            <title type="complete" ref="LIT1252Confes"/>
+                            <textLang mainLang="gez"></textLang>
+                            <note>The text has been copied from <bibl>
+                            <ptr target="bm:Ludolf1691Commentarius"/>
+                            <citedRange>237-241</citedRange>.
+                          </bibl></note>
+
+                            </msItem>
+
+                        <msItem xml:id="ms_i2">
+                          <locus from="6r"></locus>
+                          <title type="complete" ref="LIT3180Anaph" xml:lang="gez">
+                            አኰቴተ፡ ቍርባን፡ ዘቅዱስ፡ ዲዮስቆሮስ፡
+                          </title>
+                          <textLang mainLang="gez"></textLang>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                          <locus from="9v"></locus>
+                          <title type="complete" xml:lang="en">Lord's Prayer</title>
+                          <textLang mainLang="gez"></textLang>
+                        </msItem>
+                    </msContents>
+
+                    <physDesc>
+                        <objectDesc form="Notebook">
+                            <supportDesc>
+                                <support>
+                                    <material key="paper"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">9</measure>
+                                    <measure unit="leaf" type="blank">1</measure>
+                                    <dimensions type="outer" unit="in">
+                                        <height>6.125</height>
+                                        <width>3.75</width>
+                                    </dimensions>
+                                </extent>
+
+                            </supportDesc>
+
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="19 25">
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <date notBefore="1800" notAfter="1830" resp="PRS10171Wright">Of the beginning of the 19th century</date>
+                                <desc>Neatly written</desc>
+                            </handNote>
+
+                        </handDesc>
+
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#1v"/>
+                                    <desc type="ScribalSignature"></desc>
```

Comment:
I am not sure from the description whether it is a scribal signature, we do not seem to know whether this note was written by Rich or someone else.
I don't know who he is either.

Reply:
I would assume that the entire text of this small manuscript might have transcribed by Mr Rich in his early years, 
and he left a note about this.

Reply:
I will leave the category Unclear untill we could say more about the character of this note.